macOS/fill-cache.sh

```diff
@@ -0,0 +1,39 @@
+#! /bin/sh
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+set -x
+set -e
+
+WORKSPACE=$(pwd)
+MAKE=make
+
+mkdir cache
+
+cd ${WORKSPACE}/tools/kconfig-frontends
+./configure \
```

Comment:
Why not reuse cibuild.sh? I don't think there is too much difference to block sharing the same script.

Reply:
do you mean install_tools part?

Reply:
Yes, we can even detect the host dynamically and download the tool for different host if needed.

Reply:
i removed macos-specific scripts.